inc/Engine/License/API/User.php

```diff
@@ -242,7 +242,18 @@ public function get_pma_addon_btn_text( string $sku ) {
 			return '';
 		}
 
-		return $plan->button->label;
+		$label = $plan->button->label;
+
+		// Translate known button labels.
+		if ( 'Get Advanced' === $label ) {
+			return __( 'GET ROCKET INSIGHTS', 'rocket' );
```

Comment:
The CSS style on the button will transform this to uppercase.
```suggestion
			return __( 'Get Rocket Insights', 'rocket' );
```